src/load_save.c

```diff
@@ -57,6 +59,11 @@ void CheckForFlashMemory(void)
     }
 }
 
+void ClearSav3(void)
+{
+  CpuFill16(0, &gSaveblock3, sizeof(struct SaveBlock3));
```

Comment:
tabs

Reply:
I took the liberty to fix it. Hope it's fine.